spark_log_parser/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Tools for providing Spark event log"""
 
-__version__ = "0.1.1"
+__version__ = "0.1.3"
```

Comment:
minor observation, but should this be `0.1.2` instead?

Reply:
No because PR#12 will increment to 0.1.2